conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
ok. it looks like the ReorderWindow could be nicer by calling `dur.ParseNDuration` as it allows for humanfriendly syntax like `10min30s`, we use this approach already in a bunch of other options like chunkspan settings (note: this was just renamed, in the version this PR is based on it was still called `dur.dur.ParseUNsec` so i suggest you rebase on top of master)